lib/BuildSystem/SwiftTools.cpp

```diff
@@ -523,7 +525,10 @@ class SwiftCompilerShellCommand : public ExternalCommand {
       return false;
 
     // Execute the command.
-    if (!bsci.getExecutionQueue().executeProcess(context, commandLine)) {
+
+    auto handler = std::bind(&SwiftCompilerShellCommand::partialOutputAvailable, this, std::placeholders::_1);
```

Comment:
Can you just use an inline lambda here to forward? I find it more readable than `std::bind()`.